_data/sites.json

```diff
@@ -7293,9 +7293,9 @@
 
     {
         "name": "Runescape",
-        "url": "http://services.runescape.com/m=rswiki/en/Account_Help",
+        "url": "https://www.runescape.com/zendesk/support-form?form=360000041149",
         "difficulty": "hard",
-        "notes": "Under 5. Account Settings > A. Account Management > V. Account Deletion. You must mail or fax an affidavit to the Runescape office along with a copy of your legal identification",
+        "notes": "Select 'Remove your personal data and permanently disable your account' as your request type and fill out the rest of the form.",
```

Comment:
Does that mean you don't have to contact support anymore? Maybe we should lower the difficulty to medium

Reply:
Yep, fair enough. Updated.